CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 FEATURES:
 * Adds ability to limit the lifetime of service principal secrets in Azure through `explicit_max_ttl` on roles ([GH-199](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/199))
 
+## v0.19.2
+IMPROVEMENTS:
+* Updated dependencies [[GH-215]](https://github.com/hashicorp/vault-plugin-secrets-azure/pull/215)
+  * `github.com/Azure/azure-sdk-for-go/sdk/azidentity` v1.5.2 ->  v1.6.0
+  * `github.com/hashicorp/go-retryablehttp` v0.7.1 -> v0.7.7
```

Comment:
Were these updated under the PR above?

Reply:
Yup! https://github.com/hashicorp/vault-plugin-secrets-azure/pull/215/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R52